Amijakan/mochatto
# rf-358 Remove the separate state for selfUserInfo

## Which Issue was this PR made for?

- Closes #358 

## What is within the scope with this PR

Removing the separate state hook for selfuserinfo from the RoomPage.

## Testing guidance

Make sure that everything works since this was a pretty big refactor including

- Make sure moving avatar works
- Make sure mute, active toggles work
- Make sure the visualizer works on the Join page and room page
- Make sure that screen sharing works
- Make sure that works for all other users
- Maker sure that renaming works
